content/references/markdown/extensions.md

```diff
@@ -0,0 +1,130 @@
+---
+kind: reference
+topic: markdown
+prev: /references/markdown/
+---
+
+# Codewars' Markdown Extensions
+
+Codewars adds few Markdown extensions for writing kata descriptions.
+
+## Sequential Code Blocks
+
+When we have a sequence of code blocks like the following:
+
+````markdown
+```javascript
+const add = (a, b) => a + b;
+```
+```typescript
+const add = (a: number, b: number) => a + b;
+```
+```ruby
+def add(a, b)
+  a + b
+end
+```
+````
+
+Only the code block of the active language is shown.
+So if the active language was TypeScript, the above Markdown outputs:
+
+> ```typescript
+> const add = (a: number, b: number) => a + b;
+> ```
+
+This is useful when showing examples.
+
+Note that the code block must have info string set to a valid language id.
```

Comment:
```suggestion
Notes:
* The code block must have info string set to a valid language id. To find the language id, visit its documentation from the [languages page](/languages/).
* To avoid rendering errors, always put an empty line before and after the code block or the group of code bloks, while consecutive code blocks for different languages must not be separated with empty lines.
```

Reply:
Your suggestion looks good, but it includes the text from the next line so committing this will create duplicates.

You can comment on multiple lines by clicking the line number, then Shift+click the ending line. This way, you can suggest changes to multiple lines.